packages/i18n/src/locales/en.json

```
@@ -229,7 +229,10 @@
         "label": "Choose Language"
       },
       "themeToggle": {
-        "label": "Toggle Dark Mode"
+        "label": {
+          "light": "Toggle Dark Mode",
+          "dark": "Toggle Light Mode"
```

Which one is better?🤔

```suggestion
          "dark": "Activate Light Mode"
```
```suggestion
          "dark": "Toggle to Light Mode"
```

---

"Switch to"?